### DISCUSSION TITLE
Bounty Program and Rewards System

### DISCUSSION THREAD

**Discussion by jeffjenx** [RetroAchievements/RAWeb#641] [Ideas]
In Issue #638, it was suggested that there be some additional Art Badge rewards and the discussion got me thinking about ways we could incentivize our community to make improvements and help direct the team's focus on what is most important to them.

One suggestion was to use a recurring competition, similar to Achievement of the Week, but the concern was that a competition will only really benefit the winner.

Instead of a weekly competition, maybe we could do some sort of bounty program and rewards system, similar to how Stack Exchange network and Microsoft Rewards work.

As users perform tasks that improve the community, they could be given points that could then be used to make various "purchases." It doesn't have to be monetary in value, but perhaps something like using their points to change their username, add flair to their profile, enable a non-essential feature, put their own bounties on updating sets or voting for enhancements.

It is a sort of digital currency that helps drive engagement and rewards people who are making regular and helpful contributions that could be expanded upon in the future.

**Comment by Etron-RA** [comment]
The bounty program would work well to reward bigger sets with more points than a small set or a badge icon.  We could also increase the bounty if a commission is left sitting for a while.

Like the idea of "purchases", not sure what we can offer outside of cosmetic changes to your user page. Getting you name in a different colour is popular on discord. I was thinking of simply having a tiered mastery badge that changes as you get more bounties and including your bounty points as one of the stats shown on the user page.
